external/storm-kafka-client/src/test/java/org/apache/storm/kafka/spout/MaxUncommittedOffsetTest.java

```diff
@@ -57,10 +57,9 @@
     //Current tests require that numMessages >= 2*maxUncommittedOffsets
     private final int numMessages = 100;
     private final int maxUncommittedOffsets = 10;
-    //This is set to be the same as maxUncommittedOffsets since it is difficult to test maxUncommittedOffsets when maxPollRecords is not the same
-    //If maxPollRecords is larger, a single call to poll will emit more than maxUncommittedOffsets messages
-    //If maxPollRecords is lower, it will cap how far past the commit offset the spout can read when there are failed tuples ready for retry
-    private final int maxPollRecords = maxUncommittedOffsets;
+    //Current tests require that maxPollRecords is less than maxUncommittedOffsets
```

Comment:
Is this still true? If so, let's put an assertion validating this in the @Before method or something.